tests/MoBi.Tests/Core/SBML/ReactionImporterSpecs.cs

```diff
@@ -169,8 +170,8 @@ protected override void Because()
       {
       }
 
-      [Observation]
-      public void ShouldRaiseOnInvalidFileConvertion()
+      [Observation, Ignore("Test is unstable. Remove Ignore with https://github.com/Open-Systems-Pharmacology/MoBi/issues/868")]
```

Comment:
Ignoring this test for now. I opened an issue to update the SBML importer and noted this test in the issue.

Reply:
@Yuri05 FYI